### DISCUSSION TITLE
Character variant that inherits creature in another mod will try to find its sprite from wrong mod

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15235] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

mod A creates a new character which sprite path uses %ModDir% 
mod B creates a variant of that character
game tries to find sprite in mod B instead of mod A

### Reproduction steps

1. download mod "Chimera Remake"
2. try to create a variant of Chimera_head in a separate mod

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
related issue #13302

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
